python_module/task1/calc.py

```diff
@@ -0,0 +1,120 @@
+"""This module performs 4 basic math operation:
+add, subtract, multiply, divide on two float numbers
+
+"""
```

Comment:
Please align the dockstring
For example:

"""
Information
"""

or 

"""Information"""

Important to keep it on the same template